INSTALL.md

```diff
@@ -54,7 +54,7 @@ with the required dependencies:
 
     conda create -y -n compiler_gym python=3.8
     conda activate compiler_gym
-    conda install -y -c conda-forge cmake doxygen pandoc patchelf
```

Comment:
I believe this is actually needed for the bazel build (which runs cmake to compile a couple of deps), and the existing CMake instructions are compatible, because it injects the new cmake at `~/.local/bin` in the $PATH before conda's

Reply:
I added it back in.